examples/aes-example.cpp

```diff
@@ -0,0 +1,167 @@
+/*
+ * #%L
+ * %%
+ * Copyright (C) 2020 BMW Car IT GmbH
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+
+#include <mococrw/symmetric_crypto.h>
+#include <iostream>
+
+using namespace mococrw;
+
+static const std::vector<uint8_t> plaintext = utility::fromHex("deadbeef");
+static const SymmetricCipherMode aeOperationMode = SymmetricCipherMode::GCM;
+static const SymmetricCipherMode plainOperationMode = SymmetricCipherMode::CBC;
+static const SymmetricCipherPadding padding = SymmetricCipherPadding::PKCS;
+static const std::vector<uint8_t> aesKey = utility::fromHex("000102030405060708090a0b0c0d0e0f");
+static const std::vector<uint8_t> authData = utility::fromHex("beefdead");
```

Comment:
```suggestion
static const std::vector<uint8_t> associatedData = utility::fromHex("beefdead");
```

Would rename so that it's clear from the "**A**uthenticated **E**ncryption with **A**ssociated **D**ata" acronym that this is the latter part.

Reply:
yep. When I first read it, I had the impression that this should be the authTag.